x-pack/test/security_solution_cypress/cypress/e2e/explore/dashboards/enable_risk_score_redirect.cy.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  ENABLE_HOST_RISK_SCORE_BUTTON,
+  ENABLE_USER_RISK_SCORE_BUTTON,
+} from '../../../screens/entity_analytics';
+
+import { cleanKibana } from '../../../tasks/common';
+import { login } from '../../../tasks/login';
+import { visit } from '../../../tasks/navigation';
+import { clickEnableRiskScore } from '../../../tasks/risk_scores';
+import { RiskScoreEntity } from '../../../tasks/risk_scores/common';
+
+import { ENTITY_ANALYTICS_URL } from '../../../urls/navigation';
+import { PAGE_TITLE } from '../../../screens/entity_analytics_management';
+
+// FLAKY: https://github.com/elastic/kibana/issues/165644
+describe(
+  'Enable risk scores from dashboard',
+  { tags: ['@ess', '@serverless', '@brokenInServerless'] },
+  () => {
+    before(() => {
+      cleanKibana();
```

Comment:
Taking into consideration the assertions and actions we are performing here, I believe we could even remove the `cleanKibana` method.